### ISSUE TITLE
Lyrics fail on m4a files

### ISSUE THREAD

**Issue by valrus** [valrus/alfred-mpd#2]
Suspect this is due to the album name being mangled into "DAMN_" by the MacOS filesystem. The traceback:

```
[14:35:08.231] ERROR: Play with mpd[Run Script] Traceback (most recent call last):
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/ianmccowan/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.E9AAC0A8-7F64-4A19-920C-0E071F0C8807/actions/lyrics.py", line 51, in <module>
    main()
  File "/Users/ianmccowan/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.E9AAC0A8-7F64-4A19-920C-0E071F0C8807/actions/lyrics.py", line 16, in main
    tag_lyrics = audiofile.tag.lyrics
AttributeError: 'NoneType' object has no attribute 'tag'
```

**Comment by valrus** [comment]
It's actually because I had DAMN. as m4a files rather than mp3 and eyed3 only works with mp3.

**Comment by valrus** [comment]
Fixed in v2.1.2.
